src/panels/logbook/ha-logbook.ts

```diff
@@ -118,6 +118,26 @@ class HaLogbook extends LitElement {
                 ? ` (${item_username})`
                 : ``}</span
             >
+            ${!item.context_event_type
+              ? ""
+              : item.context_event_type == "call_service"
+              ? // Service Call
+                html` by service ${item.context_domain}.${item.context_service}`
```

Comment:
We should start making this translatable, so instead of creating the text in the backend give back keys so we can localize it here.

Reply:
(not for this PR)

Reply:
I was thinking about that since I didn't like adding ` by xxxx `. We need to make each platform have the ability to describe the events from `_entry_message_from_event` and then logbook external events need keys as well... anyways for another PR